extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts

```diff
@@ -0,0 +1,66 @@
+import { createNodeDescriptor, INodeFunctionBaseParams } from "@cognigy/extension-tools";
+
+
+export interface ISendAppleAuthParams extends INodeFunctionBaseParams {
+	config: {
+        attachmentId: string;
+        
+        merchant_session: object;
+        supported_networks: string[];
+	};
+}
+export const sendAppleAuthNode = createNodeDescriptor({
+	type: "sendAppleAuthRingCentral",
+	defaultLabel: "Send Apple Authenticate",
+	fields: [
+		{
+            key: "attachmentId",
+            type: "cognigyText",
+            label: "Attachment ID",
+            description: "The RingCentral Attachment ID for the image that should be used"
+        },
+        {
+            key: "response_body",
+            type: "cognigyText",
+            label: "Text",
+            params: {
+                required: false
+            }
+        },
+    sections: [
+        {
+            key: "attachment",
+            label: "Attachment",
+            defaultCollapsed: true,
+            fields: [
+                "attachmentId"
+            ]
+        }
+    ],
+	form: [
+        { type: "field", key: "Text" }
```

Comment:
I think you mean  `response_body` should be the `key` in this line.

Reply:
@r2m1 could you add the changes please? Thank you very much :)

Reply:
Should be ok now !